src/vs/workbench/parts/terminal/electron-browser/terminalInstance.ts

```diff
@@ -794,14 +799,23 @@ export class TerminalInstance implements ITerminalInstance {
 		}
 	}
 
-	// TODO: This should be private/protected
 	// TODO: locale should not be optional
-	public static createTerminalEnv(parentEnv: IStringDictionary<string>, shell: IShellLaunchConfig, cwd: string, locale?: string, cols?: number, rows?: number): IStringDictionary<string> {
+	protected createTerminalEnv(parentEnv: IStringDictionary<string>, shell: IShellLaunchConfig, cwd: string, locale?: string, cols?: number, rows?: number): IStringDictionary<string> {
```

Comment:
@Tyriar How about making createTerminalEnv non static?

Reply:
I'm all for that, the only reason it's still like that is because the tests were written that way. It would be better to do protected and inherit in a super testing class. You would probably have to block out the terminal process launching stuff from launching during tests for that to work though which might be too much work for this PR.

Reply:
In that case, I'll revert that change